src/features/pastelDB/__tests__/pastelDBLib.test.ts

```diff
@@ -0,0 +1,488 @@
+import initSqlJs, { Database, QueryExecResult } from 'sql.js'
```

Comment:
Can we mock imports, so it's actually testing only the code in queries, and nothing else? Please see the example here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/loading/__tests__/utils.test.ts. Here is the article about jest mocks: https://jestjs.io/docs/manual-mocks